### PR TITLE
APM000 change rate limit quota and spike arrest to match NFRs

### PR DESCRIPTION
**Pull request by deskelleher**
## Summary
* Routine Change

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: manifest_template.yml [hunk 1]
```diff
@@ -48,12 +48,12 @@ apigee:
...
                 quota:
                   enabled: true
-                  limit: 300
+                  limit: 6000
                   interval: 1
                   timeunit: minute
                 spikeArrest:
                   enabled: true
-                  ratelimit: 600pm # 10 requests per second
+                  ratelimit: 100ps # 100 requests per second
```
Comment (strutt) [inline manifest_template.yml:56]: We recommend that if you really want 100 requests per second then the value you should set here is `6000pm`

File: manifest_template.yml [hunk 2]
```diff
@@ -94,12 +94,12 @@ apigee:
...
                 quota:
                   enabled: true
-                  limit: 300
+                  limit: 6000
                   interval: 1
                   timeunit: minute
                 spikeArrest:
                   enabled: true
-                  ratelimit: 600pm # 10 requests per second
+                  ratelimit: 100ps # 10 requests per second
```
Comment (strutt) [inline manifest_template.yml:102]: ... and here too `6000pm`.

**Review by strutt — CHANGES_REQUESTED** [review]

File: manifest_template.yml (hunk 1 above)
Comment (deskelleher) [reply manifest_template.yml]: Thanks Ben! Changed to 6000pm and pushed. This limit is from our NFRs so we really do want it.

File: manifest_template.yml (hunk 2 above)
Comment (deskelleher) [reply manifest_template.yml]: As in my other comment

**Review by strutt — APPROVED** [review]

**Pull request merged**